activemq_xml/setup.py

```diff
@@ -4,7 +4,10 @@
 from codecs import open
 from os import path
 
-import simplejson as json
+try:
+    import json
```

Comment:
I _think_ we want this the other way around: `simplejson` would be the ideal but optional lib, while `json` should be readily available. 😅

Reply:
From a security point of view, I think a good argument could be made to prefer using `json` (1st-party) instead of `simplejson` (3rd-party). What do you think?

Reply:
That's fine with me, I get that.

So the thing about this snippet is that we'll never really get an `ImportError` on `import json` as it's part of the std library. That's why I suggested the reverse:
```
try:
    import simplejson as json
except ImportError:
    import json
``` 
If `simplejson` is available, use that, otherwise fallback to the standard json library.

Since we still don't seem to want that, then I guess you can just remove the import for `simplejson` altogether.